QualityOntology.ttl

```diff
@@ -524,7 +524,7 @@ cco:PitchOrientation rdf:type owl:Class ;
 cco:PointingOrientation rdf:type owl:Class ;
                         rdfs:subClassOf cco:SpatialOrientation ;
                         cco:alternative_label "Facing Orientation"@en ;
-                        cco:definition "A Spatial Orientation of an Object in which one or more designated components of the Object are pointed toward a specified Target."@en ;
+                        cco:definition "A Spatial Orientation of an Material Entity in which one or more of its designated components are oriented toward a specified direction."@en ;
```

Comment:
@cameronmore Please edit to 'a Material Entity' not 'an Material Entity' when you have a moment. Thanks!

Reply:
Fixed!